search-provider/eks-discovery-feed-provider.c

```diff
@@ -807,7 +807,7 @@ handle_get_recent_news (EksDiscoveryFeedDatabaseContentProvider *skeleton,
                                                      "tags-match-any", tags_match_any,
                                                      "sort", EKNC_QUERY_OBJECT_SORT_DATE,
                                                      "order", EKNC_QUERY_OBJECT_ORDER_DESCENDING,
-                                                     "limit", 5,
+                                                     "limit", 1,
```

Comment:
Hmm, actually, on second thoughts, we should keep the iteration. It is possible that an app might return zero models, in which case, we'd have to have the same amount of code to deal with that problem if we were just returning an array.